pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019. PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// +k8s:openapi-gen=true
+// TidbMonitor encode the spec and status of the monitoring component of a TiDB cluster
+type TidbMonitor struct {
+	metav1.TypeMeta `json:",inline"`
+	// +k8s:openapi-gen=false
+	metav1.ObjectMeta `json:"metadata"`
+
+	// Spec defines the desired state of TidbMonitor
+	Spec TidbMonitorSpec `json:"spec"`
+
+	// +k8s:openapi-gen=false
+	// Most recently observed status of the TidbMonitor
+	Status TidbMonitorStatus `json:"status"`
+}
+
+// +k8s:openapi-gen=true
+// TidbMonitor spec encode the desired state of tidb monitoring component
+type TidbMonitorSpec struct {
+	Clusters        []TidbClusterRef  `json:"clusters"`
```

Comment:
Does the `Clusters` means one `TidbMonitor` could monitor serveral `TidbCluster`?

Reply:
Yes